tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/Constant.java

```diff
@@ -1355,13 +1356,15 @@
    */
   @Endpoint(name = "constantOf")
   public static <T extends TType> Constant<T> create(Scope scope, T tensor) {
-    return new Constant<>(
-        scope
-            .env()
-            .opBuilder("Const", scope.makeOpName("Const"))
-            .setAttr("value", tensor)
-            .setAttr("dtype", tensor.dataType())
-            .build());
+    OperationBuilder builder = scope
```

Comment:
Why is this changed?

Reply:
Because it wasn't applying the scope correctly (missed the `scope.apply` call), control dependencies and the device were missed.  The control deps are what caused the issue here, but I ran into the device elsewhere as well.